v2/yarpcprotobuf/protoc-gen-yarpc-go/main_test.go

```diff
@@ -96,17 +115,24 @@ func getCodeGeneratorRequestBytes(t *testing.T, filename string) []byte {
 			filename,
 		},
 		ProtoFile: []*descriptor.FileDescriptorProto{
-			getFileDescriptorProto(t, "src/common/common.proto"),
 			getFileDescriptorProto(t, filename),
 		},
 	}
+	for _, d := range dependencies {
+		req.ProtoFile = append(req.ProtoFile, getFileDescriptorProto(t, d))
+	}
 
 	bytes, err := proto.Marshal(req)
 	require.NoError(t, err)
 
 	return bytes
 }
 
+// getFileDescriptorProto unmarshals the given filename into a *FileDescriptorProto.
```

Comment:
Possibly to make this less confusing, we could not use the `proto.FileDescriptor()` method here but instead inline a map of serialized filedescriptors in the code? This requires a little more work when updating the files in these tests, but makes it very explicit what happens.

Reply:
Yea that's an interesting thought. If it's fine with you, I'd prefer to update that in a follow-up. We could/should introduce something similar for testing the other plugins, so we can port that work here, too.

Reply:
I'm fine with that.